configure.py

```diff
@@ -1155,29 +1155,33 @@ def system_specific_test_config(env):
   """Add default build and test flags required for TF tests to bazelrc."""
   write_to_bazelrc('test --flaky_test_attempts=3')
   write_to_bazelrc('test --test_size_filters=small,medium')
-  write_to_bazelrc(
-      'test --test_tag_filters=-benchmark-test,-no_oss,-oss_serial')
-  write_to_bazelrc('test --build_tag_filters=-benchmark-test,-no_oss')
+
+  # Each instance of --test_tag_filters or --build_tag_filters overrides all
+  # previous instances, so we need to build up a complete list and write a
+  # single list of filters for the .bazelrc file.
+
+  # Filters to use with both --test_tag_filters and --build_tag_filters
+  common_filters = ['-benchmark-test', '-no_oss']
+  # Additional filters for --test_tag_filters beyond those in common_filters
+  addl_test_filters = ['-oss_serial']
```

Comment:
I would rename it to `test_only_filters`

Reply:
Good idea! I've renamed this variable in commit bab7d7d